CONTRIBUTING.md

```diff
@@ -0,0 +1,318 @@
+
+# Developer guide
+
+## Introduction
+
+The Scap Security Guide (SSG) consists of a collection of SCAP content for different platforms such as Red Hat Enterprise Linux, Fedora, Debian, Suse and others.
+It is created and maintained by the community, and anyone can join this effort by creating new or improving the current content.
+There are three main types of content in SSG, they are rules, defined using the XCCDF standard, checks, usually written in [OVAL](https://oval.mitre.org/language/about/) format, and remediations, that can be executed on ansible, bash, anaconda installer and puppet.
+SSG also has its own templating mechanism, allowing content writers to create models and use it to generate a number of checks and remediations.
+
+The SSG content is primarily divided by platform and it can be seen on its directory structure:
+
+> **scap-security-guide/**
+├── Chromium
+├── Debian
+├── *docs*
+├── Fedora
+├── Firefox
+├── JBoss
+├── JBossEAP5
+├── JRE
+├── OpenStack
+├── OpenSUSE
+├── RHEL
+├── RHEVM3
+├── shared
+├── SUSE
+├── Ubuntu
+├── Webmin
+└── WRLinux
+
+Except for *docs*, each directory contains checks and remediations that are useful and make sense to be used on that platform.
+The shared directory contains checks and remediations that can be used by more than one platform. It also contains some utilities, that will be covered later.
+
+## Contributing
+
+Contributions can be made for rules, checks, remediations or even utilities. There are different sets of guidelines for each type, for this reason there is a different topic for each of them.
+
+### Rules
+
+...
+// shared/references/cce-rhel-avail.txt
+
+### Checks
+
+Checks are used to evaluate a rule. They are written using a customized OVAL specification and are stored as xml files inside the *template/static/oval* directory for the desired platform.
```

Comment:
`shared/templates/static/oval` vs `RHEL/7/templates/static/oval` should be explained here IMO